pages/_document.tsx

```diff
@@ -66,6 +66,12 @@ class RealmsDocument extends Document {
           <meta name="twitter:title" content="Realms" />
           <meta property="og:type" content="website" />
           <meta name="twitter:card" content="summary" />
+
+          <script
```

Comment:
can we import these dynamically instead when the user clicks on the button?

Reply:
Reason I didnt load it dynamically in the original commit is because there's plan to support on-demand loading of Jupiter

Since the PR is targeting main.js it will always get the latest version of the script.

but I've updated it to use your code snippet, which works great.

because we support different displayModes, I needed some time to think about some details regarding integrated/widget mode.